analyzer/analyzer_test.go

```diff
@@ -255,6 +255,7 @@ func TestApplier_ApplyLayers(t *testing.T) {
 					},
 					Returns: cache.LocalImageCacheGetLayerReturns{
 						LayerInfo: types.LayerInfo{
+							ID:            "sha256:932da51564135c98a49a34a193d6cd363d8fa4184d957fde16c9d8527b3f3b02",
```

Comment:
`GetLayer` doesn't return `ID`, right? We removed insertion of `ID` to the cache.
https://github.com/aquasecurity/fanal/pull/87/files/3091476f8b8234e52a44ad799d69209305f2d6ea#r384901905

Reply:
Yes sorry – cruft.